docker-compose-dev.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
+
+services:
+
+  mongo:
+    image: mongo:3.2.19
```

Comment:
```suggestion
    image: mongo:4.2
```

Reply:
@fgalan does also orion now support mongo > 3.6?

Reply:
Yes. https://github.com/telefonicaid/fiware-orion/pull/3732 ;)

Reply:
Fixed in 49c60d3